src/index.html

```diff
@@ -7,7 +8,52 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stars</h1>
+    <section class="section">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars stars--1">
+        <span class="stars__star active"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars stars--2">
+        <span class="stars__star active"></span>
+        <span class="stars__star active"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
      </div>
      
      <div class="stars stars--1">
        <span class="stars__star active"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
      </div>
      
      <div class="stars stars--2">
        <span class="stars__star active"></span>
        <span class="stars__star active"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
```
![image](https://github.com/mate-academy/layout_stars/assets/96409563/827b8c87-3fd3-466e-8828-e92ae1e033ca)